### PR TITLE
Actions partial

### PR DESCRIPTION
**Pull request by scamelot**
actions out to a partial

listPosts enforces userID match

css adjustments to fix overlap / leaking out of frame on smaller screen sizes

### CODE REVIEW

File: views/partials/actions.ejs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<% if (user.id == blogPost.userId) { %>
+    <div id="actionsContainer">
+        <form action="/blogPost/edit/<%= blogPost._id %>" action="GET">
```
Comment (scamelot) [inline views/partials/actions.ejs]: Change to method="GET"

Comment (StevDev88) [inline views/partials/actions.ejs]: Second `action` should be a `method`.

**Review by StevDev88 — CHANGES_REQUESTED** [review]
Switch `action` to `method` in actions.ejs partial.  Otherwise, looks good!

**Review by scamelot — COMMENTED** [review]
fixed

File: views/partials/actions.ejs (hunk 1 above)
Comment (scamelot) [reply views/partials/actions.ejs]: fixed

**Review by StevDev88 — APPROVED** [review]

**Pull request merged**